app/views/branded/org_admin/templates/preferences.html.erb

```diff
@@ -0,0 +1,143 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Organizational Admin Preferences</h1>
```

Comment:
Let's check with Maria on the wording of this h1. I think she may want it to say 'Template preferences'

Reply:
I'll go with your suggestion as a start.